src/util/event-formatter.ts

```diff
@@ -17,10 +17,12 @@ export class EventFormatter {
      * @return {string}
      */
     format(event: string): string {
-        if (event.charAt(0) != '\\' && event.charAt(0) != '.') {
-            event = this.defaultNamespace + '.' + event;
-        } else {
-            event = event.substr(1);
+        if (this.defaultNamespace !== false) {
```

Comment:
this is actually nicer than my pull request https://github.com/laravel/echo/pull/58 😬